### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Rewrite Maps_Screen Reader: Narrator is not reading the contents present inside "Rewrite Maps" tab item in a meaningful sequence

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#108]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS2.4.3, MAS1.3.2

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Url Rewrite-Rewrite Maps and press enter.
6) Start Screen Reader (Narrator/NVDA).
7) Verify that screen reader is reading in such a way that meaning of the information isn't lost throughout the process..

**Actual Result:**
Narrator is not reading the contents present inside Rewrite Maps tab item in a meaningful sequence. After opening "Rewrite Maps" tab item focus should land inside the content of the Rewrite Maps i.e. focus should land on "Create Rewrite Map" but here after opening the tab focus is going to the first control of the page on "Hide Sidebar"

**Note:**
Similar issue observed in every tab items

**Expected Result:**
Screen Reader should read the information in correct order throughout the process.

**User Impact:** 
Visually Impaired Users may face difficulty in navigating through the contents and controls if screen reader does nor read the controls in proper sequence

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachments:**
[Bug_108_Focus Order is not proper inside the tab item.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323823/Bug_108_Focus.Order.is.not.proper.inside.the.tab.item.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Edge; #Narrator; #MAS2.4.3; #Keyboard

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
